### PR TITLE
Add New Leagues

### PR DESCRIPTION
**Pull request by j-stoner**
Great job on the board. I used the format you have for the espn api feeds and built out additional configs for three international soccer leagues; Ligue 1 (France), Serie A (Italy), and La Lisa (Spain) as well as WNBA and Women's College Basketball. I wanted to add men's college hockey but the underlying api for that wasn't there to populate the teams. If you want to add these, great, if you don't because it is too niche, no worries, but it was a good exercise to dig in a bit. Thanks!

### CODE REVIEW

**Comment by robbydyer** [comment]
Excellent! Thanks for the contribution. I'll build and test this out, but looks good at first glance 👍

**Comment by robbydyer** [comment]
Looks like the linter is failing. If you run `script/lint` it will show you the errors. That script will also auto-fix them for certain errors.

**Comment by j-stoner** [comment]
I had some issues getting a docker built on my pi, it seemed unhappy with the architecture options. I moved everything down to my Mac and ran the lint (and some other pieces) as well as installed docker and go. I ended up with 3 warn and the rest info. the one file, main.go ended up changing in source control but it appeared to just be blank lines when I looked at the diff, so I'm not sure what's going on but hopefully that shook it loose.

**Review by robbydyer — CHANGES_REQUESTED** [review]

File: internal/sportsmatrix/http.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/robbydyer/sports/internal/twirphelpers"
 )
 
-//go:embed assets
+// go:embed assets
```
[the remark this answers is not in the document]
Comment (j-stoner) [reply internal/sportsmatrix/http.go]: Argh, ok, I did that when I was trying to do a build.local to test out the extra leagues. I removed the space but when I try to do a build or build.local to test to make sure it still compiles, I get the following error. internal/sportsmatrix/http.go:23:12: pattern assets: cannot embed directory assets: contains no embeddable files

I made that one change and updated the repo. Sorry about all of this.

**Pull request merged**